scripts/python_scripts/generate_network_status.py

```diff
@@ -86,28 +89,90 @@ def fill_status_bar(instance):
 
 
 def generate_value_matrix():
-    network_list = get_networks_list(folder="./networks")
+    network_list = generate_network_list(nova_network_list)
     returning_array = []
     for network in network_list:
         network_data_array = []
         network_data_array.append(
             "[%s](https://explorer.subquery.network/subquery/nova-wallet/nova-wallet-%s)" % (
-                network.title(), network)
+                network.get('name').title(), network.get('name'))
         )
         prod_status, prod_commit, stage_status, stage_comit = generate_progress_status(
-            network)
+            network.get('name'))
+        network_data_array.append(network.get('features'))
         network_data_array.append(stage_status)
         network_data_array.append(prod_status)
         network_data_array.append(stage_comit)
         network_data_array.append(prod_commit)
         returning_array.append(network_data_array)
-        print('%s generated!' % network.title())
+        print('%s generated!' % network.get('name').title())
     returning_array.sort()
     increment = iter(range(1, len(returning_array)+1))
     [network.insert(0, next(increment)) for network in returning_array]
     return returning_array
 
 
+def generate_network_list(url):
+    feature_list = []
+    chains_list = send_http_request(url)
+    available_projects = get_networks_list(folder="./networks")
+    for project in available_projects:
+        with open("./networks/%s/project.yaml" % project, 'r') as stream:
+            project_data = yaml.safe_load(stream)
+        project_genesis = remove_hex_prefix(project_data.get('network').get('genesisHash'))
+        chain = [chain for chain in chains_list if chain.get('chainId') == project_genesis]
```

Comment:
As I understand, you are trying to achieve find() behavior here

`next((x for x in lst if ...), [default value])` might fit better